### DISCUSSION TITLE
usePagination not applying pagination - code same as example

### DISCUSSION THREAD

**Discussion by ror-y** [TanStack/table#2399] [General] [answered]
Hello there, I have an issue where my pagination isn't getting set despite all variables being what I expect.

https://stackoverflow.com/questions/62115344/react-table-usepagination-not-applying-pagination

I believe my code is no different to the working example here: https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/pagination

Anyone know what the issue is here?

**Comment by ayushmalviya27** [comment, marked as answer]
You're using the **rows** prop instead of **page**.
Instead of `rows.map(row....`
You need to use `page.map(...`

Don't forget to get _page_ from the useTable hook.

**Reply by ror-y** [reply]
Oh now I feel dumb. Thanks!
